src/main/java/it/gov/pagopa/admissibility/rest/anpr/AnprWebClient.java

```diff
@@ -0,0 +1,66 @@
+package it.gov.pagopa.admissibility.rest.anpr;
+
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.util.InsecureTrustManagerFactory;
+import it.gov.pagopa.admissibility.config.WebClientConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import reactor.netty.http.client.HttpClient;
+
+import javax.net.ssl.SSLException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+@Component
+public class AnprWebClient {
+    private final String stringCert;
+    private final String clientKeyPem;
+
+    private final HttpClient httpClientSecure;
```

Comment:
let's store a WebClientBuilder instead of this, filled in the constructor using:

WebClient.builder()
     .clientConnector(new ReactorClientHttpConnector(httpClientSecure))